### PR TITLE
Feature/wdp190703 8

### PR DESCRIPTION
**Pull request by danielmark0116**
Edit post-box according to the instructions from project's kanban.

### CODE REVIEW

File: src/partials/70_section-blog-posts.html [hunk 1]
```diff
@@ -15,61 +15,73 @@
...
     <div class="row">
       <div class="col-4">
-        <div class="post-box">
-          <div class="cover">
-            <div class="details">
-              <div class="date"><i class="fas fa-calendar-alt"></i> 15 Jan 2016</div>
-              <div class="comments"><i class="fas fa-comments"></i> 4 Comments</div>
+        <a href="#" class="post-box-link">
+          <span class="post-box">
+            <div class="cover">
```
Comment (matwjablonski) [inline src/partials/70_section-blog-posts.html]: you cannot use `div` inside `span`

File: src/sass/components/_post-box.scss [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+    &:hover {
+      text-decoration: none;
+      color: inherit;
+
+      .btn-post {
+        background: $primary;
+      }
+
+      h5 {
+        color: $primary !important;
```
Comment (matwjablonski) [inline src/sass/components/_post-box.scss]: don't use important

File: src/sass/components/_post-box.scss [hunk 3]
```diff
@@ -1,6 +1,23 @@
 .post-box {
   margin-bottom: 2rem;
 
+  &-link {
+    color: $text-color;
+
+    &:hover {
```
Comment (matwjablonski) [inline src/sass/components/_post-box.scss]: use hover mixin from your previous PR

wait for merge previous or cherry-pick this part to this PR

File: src/partials/70_section-blog-posts.html (hunk 1 above)
Comment (danielmark0116) [reply src/partials/70_section-blog-posts.html]: fixed

File: src/sass/components/_post-box.scss (hunk 2 above)
Comment (danielmark0116) [reply src/sass/components/_post-box.scss]: Deleted '!important! -> had to instead move this code fragment a few lines down so the title change color after hovering over it :)

File: src/sass/components/_post-box.scss (hunk 3 above)
Comment (danielmark0116) [reply src/sass/components/_post-box.scss]: ok, cherry-picked the style.scss / and this mixinf file

**Pull request merged**